markdown/3.0/en/api/ff-searchbox.api.md

```diff
@@ -8,7 +8,7 @@ ___
 | **select-onclick**&nbsp;(String) **Options**: &nbsp;"true", &nbsp;"false" (default: "false") | Determines if the contents of the search box should be selected if the inside of the box is clicked. |
 | **use-suggest**&nbsp;(String) **Options**: &nbsp;"true", &nbsp;"false" (default: "true") | Determines if a request should be sent to the suggest interface as soon as a user starts typing in a search term. |
 | **suggest-delay** &nbsp;(Number) (default: 0) | Triggers a suggest request only after the delay expired and no more input is set. When the input field changes in the delay time frame the delay will be reset. |
-
+| **value**&nbsp;(String) (default: empty) | Gets or sets the value of related HTML `<input>` element. This property cannot be set on `<ff-searchbox>` element and should only be used in custom JavaScript code. Setting value triggers `input` event on `<input>` element. Therefore suggestions are shown (if enabled). |
```

Comment:
I had to read this 3 times to understand why it cannot be set and should be set at the same time.

Maybe something like:

This property cannot be set as attribute on `<ff-searchbox>` element. Though JavaScript the value of the  related `<input>` element can be get and set. Setting the value triggers an `input` event on the `<input>` element. Therefore suggestions are shown if enabled.

Reply:
I edited the description. Please check now.